src/RichText.js

```diff
@@ -5,7 +5,8 @@ import {
     convertFromHTML,
     DefaultDraftBlockRenderMap,
     DefaultDraftInlineStyle,
-    CompositeDecorator
+    CompositeDecorator,
+    SelectionState
```

Comment:
Really minor thing, but if we're moving to all-new ES2016, presumably this can end with a comma?